lib/slack-ruby-bot-server.rb

```diff
@@ -1,13 +1,20 @@
 require 'celluloid/current'
-require 'kaminari/grape'
-require 'mongoid-scroll'
+
+if ENV['MONGO_URL'] || ENV['MONGOHQ_URI'] || ENV['MONGODB_URI'] || ENV['MONGOLAB_URI']
+  require 'kaminari/grape'
+  require 'mongoid-scroll'
+end
+
 require 'grape-swagger'
 require 'slack-ruby-bot'
 
 require 'slack-ruby-bot-server/ext'
 require 'slack-ruby-bot-server/version'
 require 'slack-ruby-bot-server/info'
-require 'slack-ruby-bot-server/models'
+
+require 'slack-ruby-bot-server/models/team_mongo.rb' if ENV['MONGO_URL'] || ENV['MONGOHQ_URI'] || ENV['MONGODB_URI'] || ENV['MONGOLAB_URI']
```

Comment:
Lots of env action here. Maybe refactor it into some global method like `SlackRubyBotServer::mongo?`.